### ISSUE TITLE
Savegames do not grey out when the map is not found

### ISSUE THREAD

**Issue by abcdefg30** [OpenRA/OpenRA#17070]
or display any other indication why nothing happens when you press the load button.

**Comment by PurityLake** [comment]
I have suggestions:
* tooltip on hover
* a symbol signifying missing map would be preferable to greying out as it is less ambiguous
* dialogue box if load is pressed or eliminate the load button altogether if clicked

What are other's opinions on this?

**Comment by dragunoff** [comment]
I think one possible route to go here is to build a savegame browser similar to the replay browser. Virtually all of the UI in the replay browser can be used for saved games - viewing the map and players, filtering the games, renaming, deleting.

**Comment by pchote** [comment]
Savegames only work in singleplayer. Why would we need to filter by players?

**Comment by PurityLake** [comment]
I agree with pchote, for single-player player filters don't count unless you want to sort by AI which I'm not sure many people would but I feel that would be a separate issue

**Comment by PurityLake** [comment]
Should there be a mix of the 3 suggestions I made? If so I'll begin work on it as soon as possible

**Comment by dragunoff** [comment]
> Savegames only work in singleplayer. Why would we need to filter by players?

Yes, most of the filters don't make sense. I was talking more about the map preview and the general layout.

**Comment by abcdefg30** [comment]
:+1: to making the save game list more similar to the replay browser.

**Comment by PurityLake** [comment]
Sorry been busy with college but I'll be hoping to get a pull request for Sunday

**Comment by PunkPun** [comment]
This was fixed by adding map preview, and also allowing to donload the missing map if possible
